crates/solidity/outputs/cargo/crate/src/generated/language.rs

```diff
@@ -16,41 +16,52 @@ pub use super::{
 const DEBUG_ERROR_MERGING: bool = false;
 
 impl ParseError {
-    pub(crate) fn new<T: Into<String>>(position: TextPosition, expected: T) -> Self {
+    #[allow(dead_code)]
+    pub(crate) fn new_at<T: Into<String>>(position: TextPosition, expected: T) -> Self {
+        Self {
+            range: Range {
+                start: position,
+                end: position,
+            },
```

Comment:
suggestion for these places and other, using the shorter syntax for compactness:

```suggestion
            range: position..position,
```